packages/expo-asset/src/AssetRegistry.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ * @format
+ */
+'use strict';
+
+export type PackagerAsset = {
```

Comment:
We are going to need a bunch of things that RNWeb doesn't provide so probably don't want to include these here but instead alias them to reference some other lib.